src/components/inputcomponents/ImageInput/SingleInput.js

```diff
@@ -35,6 +54,16 @@ const singleInput = (props) => {
               </div>
             </div>
           </Dropzone>
+          <DropboxChooser appKey={APP_KEY}
+                      success={(files) => onSelect(files)}
+                      multiselect={false} 
+          >
+                     
+            <a className="ui blue button">
+              <span>Upload from dropbox</span>
+            </a>
+            <div className="dropbox"></div>
```

Comment:
This line evokes an ESLint warning. Please resolve that. New builds fail otherwise. We will soon be introducing [Prettier](https://github.com/prettier/prettier) in the project, once this pull request goes through. Maybe you can add it?